extension/src/project/projectTree.ts

```diff
@@ -123,7 +123,7 @@ export class ProjectTreeProvider implements vscode.TreeDataProvider<DisplayNode>
     public updateData(newRootNode: ProjectNode) {
         this.rootNode = newRootNode;
         vscode.commands.executeCommand("setContext", "autolisp.hasProject", ProjectTreeProvider.hasProjectOpened());
-        this.onChanged.fire();
+        this.onChanged.fire(this.rootNode);
```

Comment:
I think this is the fix for complaining from higher typescript

Reply:
Also want to share that these higher versions of TypeScript apparently changed getters/setters. From what I am reading it mostly affects inherited classes trying to use the base class getters/setters so I don't believe our code will be affected.